### ISSUE TITLE
Error with sending email

### ISSUE THREAD

**Issue by Grantglass** [anvc/scalar#168]
system/application/libraries/SendMail.php:109
 
Looking at that file, it actually looks like there is a discrepancy between the instructions in the config file and the sendmail code.
 
That function starts with:
    private function send($arr=array()) {
 
        $mail = new PHPMailer;
 
        $smtp_host = $this->CI->config->item('smtp_host');
        if (!empty($smtp_host)) {
 
Note that it’s instantiating a new PHPMailer object, and that’s what is being defined. That object also is pulling the $smtp_host from the config section, which is defined by the environment variable SCALAR_SMTP_HOST - This makes it look like the config instructions and the code are not in alignment.

**Comment by craigdietrich** [comment]
Hi @Grantglass,

Thanks for reaching out! I admit that I'm not sure what problem you're describing? Could you elaborate?

**Comment by Grantglass** [comment]
I am having trouble sending emails, my server uses php mail and I’ve tried
to configure scalar and it to send emails but it fails. How should I
configure the scalar environment variables so I can use the php mail
service and not it?

**Comment by craigdietrich** [comment]
Right, so, the way we implemented the email system is using SMTP, and 'new PHPMailer' is an SMTP helper class unrelated to PHP's mail() function.  You'd need to have SMTP active on your server and an account (e.g., a Gmail account or an institutional account) to point the SMTP fields towards.

There is no support for mail(), though that doesn't mean we can't add it, say with a field in local_settings.php that lets you choose SMTP vs mail().  I can add this to our list, but it might take a little while.
